test/unit/remote_tests.rb

```diff
@@ -8,14 +8,16 @@ module Dk::Remote
   class UnitTests < Assert::Context
     desc "Dk::Remote"
     setup do
-      @hosts    = Factory.hosts
-      @ssh_args = Factory.string
-      @cmd_str  = Factory.string
+      @hosts         = Factory.hosts
+      @ssh_args      = Factory.string
+      @host_ssh_args = { Factory.string => Factory.string }
```

Comment:
@jcredding this actually revealed a couple of mistakes in the tests - thx again!